api/v1/installation_types.go

```diff
@@ -660,3 +665,18 @@ type CertificateManagement struct {
 	// +optional
 	SignatureAlgorithm string `json:"signatureAlgorithm,omitempty"`
 }
+
+// ConditionTypes describes Installation CR's status condition type.
+// One of: Ready, Degraded, Progressing.
+type ConditionType string
+
+const (
+	// ConditionTypeReady indicates that the component is healthy.
+	ConditionTypeReady ConditionType = "Ready"
+
+	// ConditionTypeDegraded indicated that the component is not operating as desired and user action is required..
+	ConditionTypeDegraded ConditionType = "Degraded"
+
+	// ConditionTypeProgressing indicates that the component is in the process of being installed or upgraded
```

Comment:
Please end with a sentence.

All comments in this directory will be used in auto-generated docs on our website. They will also be part of the CRDs that users install. For example, see here: 
- https://docs.tigera.io/reference/installation/api
- https://projectcalico.docs.tigera.io/reference/installation/api

Reply:
Done